djangoplicity/mailinglists/models.py

```diff
@@ -128,7 +128,7 @@ def _get_mailman(self):
         return MailmanList(name=self.name, password=self.password, main_url=self.base_url)
     mailman = property(_get_mailman)
 
-    def subscribe(self, subscriber=None, email=None, async=True):
+    def subscribe(self, subscriber=None, email=None, is_async=True):
```

Comment:
@EdisonArango, I renamed the `async` variable to `is_async` because `async` is a reserved word in Python 3.7 but I'm not sure if this would break the packages that use this. So let me know what you think about renaming it. Thanks

Reply:
It's a nice solution 👍